### PR TITLE
Not all registries have stricter rules

### LINKED ISSUE #4385: Transport: IANA section inconsistency regarding 0-63 range for 62-bit values
Section 22.1.4:

   "For
   instance, the registries for 62-bit codepoints in this document have
   stricter policies for codepoints in the range from 0 to 63."

Looking then at 22.2 it states:

   The "QUIC Transport Parameters" registry governs a 62-bit space.
   This registry follows the registration policy from Section 22.1.
   Permanent registrations in this registry are assigned using the
   Specification Required policy ([RFC8126]).

No mention of the special registration rule for values in the 0-63 range. 

Section 22.3 and 22.4 do have these special rules. 

So should 22.1.4 be changed to state "may"/"sometime" have this rule. Or should a special rule be added to 22.2 and the version registry?

### PR DESCRIPTION
**Pull request by martinthomson**
Closes #4385.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -7328,12 +7328,12 @@ permanent registrations.
 
 The creation of a registry MAY identify a range of codepoints where
 registrations are governed by a different registration policy.  For instance,
-the registries for 62-bit codepoints in this document have stricter policies for
-codepoints in the range from 0 to 63.
+the frame type registry in {{iana-frames}} has a stricter policy for codepoints
```
Comment (gloinul) [inline draft-ietf-quic-transport.md:7331]: The transport error also have this range, but then have an additional range so I think this text need tweaking.

Comment (martinthomson) [reply draft-ietf-quic-transport.md:7331]: Not sure that I follow.  Frame types have a stricter policy for 0..=63.  I don't see why would need to talk about the other ~2^62 values.

**Review by janaiyengar — APPROVED** [review]

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (janaiyengar) [reply draft-ietf-quic-transport.md:7331]: I agree. @gloinul ?

**Comment by janaiyengar** [comment]
@gloinul: You have a comment here pending. I'm planning to merge this PR in ~15 hours from now, so please say something now if you still would prefer a change.

**Pull request merged**